src/devices/Nmea0183/samples/NmeaSimulator/NmeaSimulator.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>$(DefaultSampleTfms)</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Default sample TFMs is already net6.0, so I don't believe this is needed:

[https://github.com/dotnet/iot/blob/8be21fcd2c691bad349878592fce9fc726970358/src/devices/Directory.Build.props#L9](https://github.com/dotnet/iot/blob/8be21fcd2c691bad349878592fce9fc726970358/src/devices/Directory.Build.props#L9)

Reply:
True. I guess this change was done before we changed the sample TFM